docs-src/modules/ROOT/pages/core/parsers.adoc

```diff
@@ -1,8 +1,180 @@
 = Parsers
 
+Parsers are the core of what Pathom does. This section explains what Pathom parser does,
+explain their responsibilities and details how they work.
+
+== The parser job
+
+Every EQL transaction is a vector, this means that by definition an EQL transaction is
```

Comment:
Every EQL transaction is a vector, this means that, by definition, an EQL transaction is